Resources/views/Language/list.html.twig

```diff
@@ -0,0 +1,94 @@
+{% extends "eZPlatformUIBundle::pjax_admin.html.twig" %}
+
+{% trans_default_domain "language" %}
+
+{% block choice_widget %}
+    {% for child in form %}
+        {% set language = child.vars.label %}
+
+        <tr class="ez-selection-table-row">
+            <td>
+                {{ form_widget(child, {'disabled': not language.canDelete, 'attr': {'class': 'ez-selection-table-checkbox'} }) }}
+            </td>
+            <td><a href="{{ path( 'admin_languageview', {'languageId': language.id} ) }}">{{ language.name }}</a></td>
+            <td><a href="{{ path( 'admin_languageview', {'languageId': language.id} ) }}">{{ language.identifier }}</a></td>
+            <td class="ez-table-data-id"><a href="{{ path( "admin_languageview", {"languageId": language.id} ) }}">{{ language.id }}</a></td>
+            <td>
+                <a href="{{ path('admin_languageedit', {'languageId': language.id}) }}" class="pure-button ez-button{% if not language.canEdit %} pure-button-disabled{% endif %}" data-icon="&#xe606;">{{ 'language.edit'|trans }}</a>
```

Comment:
This is a link, so adding `pure-button-disabled` is not enough as this won't prevent the browser to follow the link if one clicks on it. (NB: template for section is wrong too).
See [ContentTypeGroups list template for a good example](https://github.com/ezsystems/PlatformUIBundle/blob/master/Resources/views/ContentType/list_content_type_groups.html.twig#L38-L42)